### ISSUE TITLE
Add slot for inventory PUTting

### ISSUE THREAD

**Issue by Cat121** [Valandur/Web-API#105]
Modifying inventories is awesome but it doesn't take slot index in account.

When I GET player inventory and then PUT it back then all items are still there but they are sorted from the very first slot in the inventory and not in the action bar.

Maybe it would be possible to add a "slot" option ?

```
{
"inventory": [{
      "type": {"id": "minecraft:stone"},
	  "quantity": 2
	  },{
      "type": {"id": "minecraft:tipped_arrow"},
      "quantity": 1
	  }, {
      "type": {"id": "minecraft:enchanted_book"},
      "quantity": 1
	  }]
}
```
Becomes : 
```
{
"inventory": [{
      "type": {"id": "minecraft:stone"},
	  "quantity": 2,
	  "slot": 5
	  },{
      "type": {"id": "minecraft:tipped_arrow"},
      "quantity": 1,
	  "slot": 8
	  }, {
      "type": {"id": "minecraft:enchanted_book"},
      "quantity": 1,
	  "slot": 0
	  }]
}
```

Edit : here is what I mean http://wiki.vg/File:Inventory-slots.png

**Comment by Valandur** [comment]
Implemented in v5.2.0

**Comment by Cat121** [comment]
Could you avoid empty slots when GETting a player/entity ? There are a lot of slots 😅 

Not when PUTting because it can be useful to erase a given slot.

I can also filter when receiving request.
